BUILDING.md

```diff
@@ -61,6 +61,7 @@ CMake by default, but it can be obtained from the Bullseye Backports
 repository:
 
 ``` shell
+apt-get install software-properties-common
```

Comment:
Thanks @dannenberg for catching this, it is required for the `apt-add-respository` command below.